### DISCUSSION TITLE
Increase length of Collecting Event `StationFieldNumber`

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5767] [Feature Requests]
>We have run into issues where it would be preferred to have a longer character length for the “Field Number” field in the Collecting Event Table.  Is it possible in a future update to increase the length of the field?  It is currently at 50 characters.

**Requested By:** Kesler at the San Diego Natural History Museum

**Comment by g1rly-c0d3r** [comment]
linked in https://github.com/specify/specify7/issues/8187
